libr/core/cmd_info.c

```diff
@@ -1255,6 +1255,19 @@ static int cmd_info(void *data, const char *input) {
 					break;
 				}
 				bool fullGraph = true;
+				if (input[2] == ' ' && strcmp(input + 3, "")) {
```

Comment:
what's exactly what you wanna check here? if there's a space and not a null byte after it?
```suggestion
				if (input[2] == ' ' && input[3]) {
```

then this code is simpler and more readable, but what if there are many spaces? you shoul dbetter do something like this:

```suggestion
				const char *match = r_str_trim_head_ro (input + 2);
				if (*match) {
```